library/src/main/java/com/joom/grip/io/DirectoryFileSource.kt

```diff
@@ -16,9 +16,14 @@
 
 package com.joom.grip.io
 
+import com.joom.grip.commons.toAbsoluteNormalized
 import java.io.File
+import java.nio.file.Path
+
+class DirectoryFileSource(directoryPath: Path) : FileSource {
+
+  private val directory = directoryPath.toAbsoluteNormalized().toFile()
```

Comment:
And here I'm not sure you have to normalize the path.

Reply:
rolled back as well